ios/FluentUI/Core/UIKit+SwiftUI_interoperability.swift

```diff
@@ -16,18 +16,14 @@ struct UIViewAdapter: UIViewRepresentable {
     }
 
     func makeUIView(context: Context) -> UIView {
-        return makeView()
-    }
+        let view = makeView()
 
-    func updateUIView(_ view: UIView, context: Context) {
-        // This logic should be removed wnce the "wrapping in a UIStackView" workaround is removed.
-        guard let stackView = view as? UIStackView else {
-            return
-        }
+        // Ensures that the UIView respects the frame defined in the SwiftUI layer.
+        view.setContentCompressionResistancePriority(.defaultLow, for: .vertical)
+        view.setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
 
-        for view in stackView.arrangedSubviews {
-            view.removeFromSuperview()
-        }
-        stackView.addArrangedSubview(makeView())
+        return view
     }
+
+    func updateUIView(_ view: UIView, context: Context) { }
```

Comment:
Does updateUIView still do anything? Or are we just not removing it in this review to reduce changes to the API?

Reply:
It does not do anything. It's meant to communicate changes back to the UIView when anything changes in the SwiftUI environment. The func needs to be here just because it's part of the requirements of the UIViewRepresentable protocol.